recipes/elfio/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
```

Comment:
```suggestion
conan_basic_setup(TARGETS)

find_package(elfio REQUIRED CONFIG)

add_executable(example example.cpp)
target_link_libraries(example elfio::elfio)
```

Reply:
nice, I assume you tested that this works, yes?

Reply:
but why this way, there are 3 ways to link, ${CONAN_LIBS}, CONAN_PKG::elfio and elfio::elfio
there might be even more
and we can not use all, so why that and not any other

Reply:
all these ways are fine. but this one is the best for transparent integration.

Reply:
actually, I think I like CONAN_PKG::elfio best, 
this is also what I mostly use, 
to be sure it's not found outside of conan, what could happen with elfio::elfio , right? 

so to test the package, I think ${CONAN_LIBS}, CONAN_PKG::elfio  are better suited

Reply:
> to be sure it's not found outside of conan, what could happen with elfio::elfio , right?

No